libpod/reset.go

```diff
@@ -77,18 +77,35 @@ func (r *Runtime) Reset(ctx context.Context) error {
 		}
 	}
 
+	xdgRuntimeDir := os.Getenv("XDG_RUNTIME_DIR")
```

Comment:
The documentation `XDG_RUNTIME_DIR` says:
```
 If $XDG_RUNTIME_DIR is not set applications should fall back to a replacement directory with similar capabilities and print a warning message. Applications should use this directory for communication and synchronization purposes and should not place larger files in it, since it might reside in runtime memory and cannot necessarily be swapped out to disk.
```

So this should probably fallback to `/run/user/<uid>`.